index.html

```diff
@@ -917,15 +917,30 @@ <h2>Extensibility</h2>
         <h2>Terms of Use</h2>
         <p>Terms of use can be utilized by an <a>issuer</a>, <a>subject</a> or a
         <a>holder</a> to express limitations on the use of information
-        expressed by the Verifiable Credentials Model. The expression of
-        terms of use are performed via the following <a>property</a>:
+        expressed by the Verifiable Credentials Model. The <a>issuer</a> places 
+       the terms of use
+       inside the credential before it is converted into a verifiable credential.
+       The <a>holder</a> places the terms of use inside the <a>profile</a> before it is
+       converted into a <a>verifiable profile</a>.</p>
+	<p>
+       Note: it is for further study how a subject who is not a holder places terms of
+       use on his  or her verifiable credentials. One way could be for the subject to
+       request the issuer to place them inside the issued verifiable credentials. Another
+       way could be by the subject delegating a verifiable credential to a holder and
+       placing terms of use restrictions on the delegated verifiable credential.
+</p>
+       <p> The expression of
+        terms of use may be performed via the following <a>property</a>:
         </p>
 
         <dl>
           <dt><var>termsOfUse</var></dt>
           <dd>The value of this property MUST be one or more terms of use
-            descriptions that provide enough information to a <a>verifier</a>
-            to determine how they may utilize the given information.
+            that provide enough information to a <a>verifier</a>
+            to determine how they MUST utilize the given information.
```

Comment:
The first MUST is defining what the content of the VC must contain, whilst the second MUST is saying how the verifier must behave. So they are both needed as they apply to different things.
It is possible to test the second must by carefully crafting a terms of  use that can be externally measured. E.g. it could be 'transaction must be less than $100', then two transactions are sent to the verifier using the same VC, one for <100 and one >100.  The second one must be rejected whilst the first could be accepted or rejected. But one could think of lots of others as well e.g. Subject must be notified by email when xxx happens, then perform a test that does xxx and check that the subject receives an email.

Reply:
At the risk of stepping into a long conversation without the full background, I will comment that the way in which this MUST is worded is not testable.  To see this, try to reword the MUST above in the form 'The verifier MUST xyz'.  I would suggest that this MUST be changed to 'are to'.  You can then provide additional normative MUST statements as necessary, rather than as part of a parenthetical.  My comment is purely structural, not an attempt to read on whether normative statements are appropriate.

Reply:
@burnburn How about this wording
 "The value of this property MUST be one or more terms of use that tell the verifier what actions it MUST perform if it is to accept the verifiable credential or verifiable profile. If the verifier is not willing to accept these terms of use then it MUST reject the verifiable credential or verifiable profile."